gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -173,7 +173,9 @@ private static void configureErrorProneOptions(
                 "CatchSpecificity",
                 "InlineMeSuggester",
                 "PreferImmutableStreamExCollections",
-                "UnusedVariable");
+                "UnusedVariable",
+                // We don't use 'var' yet
+                "Varifier");
```

Comment:
more explicitly, https://github.com/palantir/gradle-baseline/blob/develop/baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/VarUsage.java throws `error: [VarUsage] The `var` keyword results in illegible code in most cases and should not be used.` so conflicts with `Varifier`

Reply:
:+1: good call, I've described the rationale in the comment